app/assets/stylesheets/show.scss

```diff
@@ -0,0 +1,76 @@
+@import "foundation_and_overrides";
+@import "index";
+
+.art-details {
+  margin: auto;
+  margin-top: 20px;
+  margin-bottom: 35px;
+  display: block;
+  width: 50%;
+  background-color: #088da5;
+  box-shadow: 5px 5px 3px black;
+  li {
+    list-style-type: none;
+    font-size: 1.5em;
+    color: white;
+    text-align: center;
+  }
+}
+
+.votes {
+  margin: 7.5px 5px 0;
+  background-color: #088da5;
+  color: white;
+}
+
+.review {
+  background-color: lighten(#088da5, 60%);
+  border-radius: 10px;
+  .review-content {
+    list-style-type: none;
+    width: 70%;
+  }
+  .user-specific-content {
+    list-style-type: none;
+    margin-right: 5px;
+    li {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3